wire-library/wire-tests/src/jvmKotlinInteropTest/proto-kotlin/com/squareup/wire/protos/kotlin/NoFields.kt

```diff
@@ -10,13 +10,15 @@ import com.squareup.wire.ProtoWriter
 import com.squareup.wire.Syntax.PROTO_2
 import kotlin.Any
 import kotlin.Boolean
+import kotlin.Deprecated
```

Comment:
Didn't know Kotlin has its own Deprecated now, nice!

Reply:
We already use it for fields. The downside with this one is that it requires a message.

Reply:
Oh, wait, I'm dumb, I confused it with `Nullable`... NVM